1040.rb

```diff
@@ -0,0 +1,26 @@
+n1 = gets.to_f
+n2 = gets.to_f
+n3 = gets.to_f
+n4 = gets.to_f
+
+media = (2*n1 + 3*n2 + 4*n3 + 1*n4)/10
+mediar = media.round(1)
+puts "Media: #{mediar}"
+
+if mediar >= 7.0
+    puts "Aluno aprovado."
+elsif mediar < 5.0
+    puts "Aluno reprovado."
+else 
+    puts"Aluno em exame."
+    exame = gets.to_f
+    media_final = (mediar + exame)/2
+    media_finalr = media_final.round(1)
+    if media_finalr < 5.0
+        puts "Aluno reprovado."
```

Comment:
> For these 2 cases (approved or reproved after the exam) print the message "Media final: " (Final average) followed by the final average for this student in the last line.

Você deve colocar também quando o aluno é reprovado qual foi a média final arredondada. 

```suggestion
        puts "Aluno reprovado."
        puts "Media final: #{media_final_arredondada}"
```